HW_1/task_1.py

```diff
@@ -0,0 +1,13 @@
+# Найдите сумму цифр трехзначного числа.
+
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+Number = int(input('Введите трехзначное число: '))
+ 
+if Number > 99 and Number < 1000:
+    print ('Сумма чисел равна: ' , Number // 100 + (Number // 10) % 10 + Number % 10) 
```

Comment:
У операций деления одинаковый приоритет.
```suggestion
    print ('Сумма чисел равна: ' , Number // 100 + Number // 10 % 10 + Number % 10) 
```